src/azul/indexer/__init__.py

```diff
@@ -93,12 +93,12 @@ def parse(cls, prefix: str) -> 'Prefix':
         >>> p = Prefix.parse('a')
         Traceback (most recent call last):
         ...
-        ValueError: ('Missing partition prefix length.', 'a')
+        ValueError: ('Missing partition prefix length', 'a')
 
         >>> Prefix.parse('aa/')
         Traceback (most recent call last):
         ...
-        azul.RequirementError: ('Prefix source cannot end in a delimiter.', 'aa/', '/')
+        azul.RequirementError: ('Prefix source cannot end in a delimiter', 'aa/', '/')
```

Comment:
I didn't realize that some existing error messages did end with a period. I think if we were to do this properly, we'd need to remove all the periods in a separate commit preceding the main commit. Probably not worth the bother of having Hannes review again.

So, sorry for the bother, but I think we should revert most of this fixup, maybe keeping the removal of the period on line 124/129 since it's a new edition rather than a modification of an existing one. And the new doctest is probably good. But let's avoid these other complications to the diff.

Reply:
Spoke with Noah over a call, decided to append a PR to the branch that removes the extra periods within the class.

Reply:
When I was adding the missing test it seemed better to track that addition within its own commit. Let me know if you'd like me to squash it down.